backend/tests/hmc-course.test.ts

```diff
@@ -78,4 +77,47 @@ describe("src/hmc-api/course.ts", () => {
             } as APIv4.SectionIdentifier),
         ).toEqual("SP2023P1");
     });
+
+    test("parseBoomiEquivalent", () => {
+        const boomi_courses = parseCourseBoomi(BOOMI_DB_SAMPLE);
+        console.log(JSON.stringify(boomi_courses));
+        expect(boomi_courses).toStrictEqual(BOOMI_DB_EXPECTED);
+    });
 });
+
+const PARSED_SAMPLE_PATH = "src/hmc-api/sample/parsed-sample-v4.json";
+
+const BOOMI_DB_SAMPLE = `
+DBSTART|85ba3f56-d039-43dd-ad22-913c0e2b42b8|2|@|BEGIN|2|@|OUT_START|3|@|SMPL010A EX|^|Example Class|^|SAFR|^||^|SC|^|Example Description
+|#|SMPL127  EX|^|Another Example Class|^|SAFR|^|05.0201|^|PZ|^|Indeed, another example description
```

Comment:
can we modify the test case so that one of the test data is not newline-separated from the next one?